core/arch/opnd.h

```diff
@@ -845,7 +845,7 @@ enum {
 #    ifdef AARCH64
     DR_NUM_SIMD_VECTOR_REGS = DR_REG_Z31 - DR_REG_Z0 + 1,
 #    else
-    DR_NUM_SIMD_VECTOR_REGS = 0,
+    DR_NUM_SIMD_VECTOR_REGS = DR_REG_Q31 - DR_REG_Q0 + 1,
```

Comment:
Q15 not Q31.  (This is where a STOP would be useful.)  There are 16 128-bit SIMD regs on ARM.  See the comment next to DR_REG_Q16.

Reply:
Ok, I see. This is very arbitrary for ARM. We chose the SVE registers for ARMv8. What about NEON etc.? I wonder if we should not define for ARM at all and there should be more distinct names specific to ARM/AArch64? I've added a comment.

Reply:
This *is* NEON, as explained.  This is not arbitrary, as discussed offline.  The only question is whether a user would also want the number of 64-bit D or 32-bit S registers, which is 32 instead of 16.  Adding or distinguishing those could be in a comment for the future.